## Hi! I am Florian - Nice to meet you ðŸ‘‹

:computer: Iâ€™m currently developing scalable workflows and tools to analyze and process spatial omics and single-cell multi-omics data.

## :fire: Key Projects

### Tools
- A tool to evaluate and classify predicted peak profiles to assess the binding specificity of a protein to its targets: [StoatyDive](https://github.com/heylf/StoatyDive).
- Lightweight, modular biosample management platform designed to streamline centralized research data handling and sample tracking within biomedical research projects: [LiMetrack](https://bitbucket.org/schwarzlab/limetrack).
- There are more private projects (please ask me about them).

### Workflows
- Nextlfow Xenium data processing pipeline: [nf-core/spatialxe](https://nf-co.re/spatialxe/dev/).
- Spatial omics gene pane design workflow: [Galaxy spatial Omics Marker gene detection workflow](https://usegalaxy.eu/published/workflow?id=9adf0402b641042c).
- A webserver to process, analyse and visualise CLIP-Seq data: [Galaxy CLIP Explorer](https://usegalaxy-eu.github.io/index-clip.html).
- Galaxy ATAC-Seq data processing workflow: [ATAC-Seq workflow](https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/atac-seq/workflows/main_workflow.html), together with the Galaxy ATAC-Seq training material [ATAC-Seq data analysis](https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/atac-seq/tutorial.html).
- Galaxy CUT&RUN data processing worfklow: [CUT&RUN workflow](https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/cut_and_run/workflows/main_workflow.html), together with the Galaxy CUT&RUN training material [CUT&RUN data analysis](https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/cut_and_run/tutorial.html).
- There are more private projects (please ask me about them).

### Other Projects that I am supporting

- A standardized Nextflow pipeline for benchmarking variant-calling workflows, enabling reproducible and FAIR analysis: [nf-core/variantbenchmarking](https://github.com/nf-core/variantbenchmarking).
- Bioinformatics best-practice analysis pipeline for processing 10x Genomics single-cell RNA-seq data: [nf-core/scrnaseq](https://github.com/nf-core/scrnaseq/tree/2.6.0).
- A flexible framework for applying spatially aware clustering methods: [SACCELERATOR](https://github.com/SpatialHackathon/SACCELERATOR).
- Nextflow pipeline for processing highly-multiplexed imaging data, as produced by technologies such as Cycif, MIBI, CODEX, SeqIF among others: [nf-core/mcmicro](https://github.com/nf-core/mcmicro).
- There are more private projects (please ask me about them).

## :mortar_board: Scientific Interests:
- Image-based spatial transcriptomics
  - Artefact detection
  - Quality control
  - Noise estimation
  - Standard data processing
- Single cell multiomics
  - Standardized data processing
  - Demultiplexing
  - Quality control
  - Artefact detection
- Bioinformatics pipelining
  - Nextflow, Galaxy, Snakemake
  - Scaling
  - Deploying workflows
  - resource and parameter optimization

## :seedling: Iâ€™m also exploring:
- Deep learning methods for artifact detection
- GPU-accelerated data processing
  
## :mailbox_with_no_mail: How to reach me:
- Email: florian.heyl@dkfz-heidelberg.de
- LinkedIn: https://www.linkedin.com/in/florian-heyl/

## :bar_chart: GitHub Stats:
![](https://github-readme-stats.vercel.app/api/top-langs/?username=heylf&theme=dark&hide_border=false&include_all_commits=false&count_private=false&layout=compact)


 
